ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcConfigKeys.java

```diff
@@ -279,6 +279,18 @@ static boolean heartbeatChannel(RaftProperties properties) {
     static void setHeartbeatChannel(RaftProperties properties, boolean useSeparate) {
       setBoolean(properties::setBoolean, HEARTBEAT_CHANNEL_KEY, useSeparate);
     }
+
+    String EXCEPTIONAL_WARN_LOG_BATCH_DURATION_KEY = PREFIX + ".exceptional-warn-log.batch.duration";
+    TimeDuration EXCEPTIONAL_WARN_LOG_BATCH_DURATION_DEFAULT = TimeDuration.ZERO; // do not batch
```

Comment:
Let's make the default `5s`?

Reply:
Sure, we can have the batch option default on.